packages/core/src/components/Breadcrumbs/Breadcrumb.tsx

```diff
@@ -24,6 +24,8 @@ export type Props = {
   onClick?: () => void;
   /** Mark the breadcrumb as selected. */
   selected?: boolean;
+  /** Adds a data-tracking-name attribute */
```

Comment:
small nit for these prop descriptions throughout the pr: end with a `.` to be consistent